### ISSUE TITLE
Bug: Appears to have trouble with forked repos

### ISSUE THREAD

**Issue by hannahhoward** [carbonfive/funnel#29]
See my PR. I'm pretty sure it's up to date but I think it's failing Funnel because I forked the repo

**Comment by outofambit** [comment]
Yes this true, and something that should be fixed. The cause is that a `Scent` only captures the user and repo name of the pushed to branch. This works fine for branches in the same repo, but not those in different ones.

I think the solution to this is to modify https://github.com/carbonfive/funnel/blob/5cc5c05b3ff186538f2a9de959502c5027d87bf5/lib/funnel/scent.ex#L46 to capture this extra information from the GitHub webhook and then utilize it in https://github.com/carbonfive/funnel/blob/master/lib/funnel/investigator/strategy.ex
